cpp/src/structs/structs_column_view.cpp

```diff
@@ -36,7 +36,9 @@ column_view structs_column_view::get_sliced_child(int index) const
                      size(),
                      child(index).head<uint8_t>(),
                      child(index).null_mask(),
-                     child(index).null_count(),
+                     // TODO: could potentially compute the actual count here, but at
+                     // the moment this interface doesn't take a stream.
+                     UNKNOWN_NULL_COUNT,
```

Comment:
Would it be incorrect to lean into this and remove the `TODO`?

Reply:
You mean change the interface to take a stream?  I think I'd be inclined to do that as a followup PR.

Reply:
Sorry, I meant that we remove the `TODO`, and leave it at `UNKNOWN_NULL_COUNT`.

On second thought, maybe we should leave the `TODO` in as a reminder to revisit.